components/dialog/README.md

```diff
@@ -258,6 +258,86 @@ document.querySelector('#open').addEventListener('click', () => {
 });
 ```
 
+## Loading Asynchronous Content
+
+The `until` directive can be used for manging the rendering of dialog content when it is loaded asynchronously. The example below loads some dialog content asynchronously, displaying a [Loading Spinner](../loading-spinner/) until the content is ready.
+
+Notes on this example:
+- Dialog sizing: `this.resolveLoadingComplete` (part of `LoadingCompleteMixin`) on image load triggers the dialog resize
+- Focus management: focus will go to the first footer button by default. If focus should instead go to a focusable element in the content then that would need to be specified.
```

Comment:
> If focus should instead go to a focusable element in the content then that would need to be specified.

Does this mean handled by them buy calling `focus()` on the element they want focused, or does the `autofocus` attribute also work? Maybe worth clarifying, though it looks like the example is calling `focus()`.

Reply:
The chrome dev console has a message "Autofocus processing was blocked because a document already has a focused element." when I add `autofocus` to the `d2l-button` and focus remains on the primary dialog footer button.